### PR TITLE
updated HU transcription

### PR DESCRIPTION
**Pull request by enerbewow**

### CODE REVIEW

**Comment by enerbewow** [comment]
yo. I hope I'm not to late

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:157]: Fix indentation.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:33]: We changed to `Használd a` instead of `"Írd be`. Does it not apply here?

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:313]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:317]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:329]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:333]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:337]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:353]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:357]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:9]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:17]: Remove extra spaces.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:502]: Missing closing bracket: `}` on next line.

**Review by nickdnk — CHANGES_REQUESTED** [review]
Thanks. Can you fix these things and answer questions?

**Comment by enerbewow** [comment]
oh shit. I used notepad++ compare and copied the lines.. I thought I fixed all the bad spaces and tabulators :D

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Thanks for the mention. I did this at night and i changed it somehow.
btw. "Használd a"  meaning - "Use the", "Írd be" meaning - Type in something

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Fixed

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Fixed

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Fixed

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Fixed

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Fixed

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Fixed

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Fixed

**Comment by enerbewow** [comment]
I go through all the lines again. and fixed every spaces

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Fixed

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt:157]: I don't see what you meaning here.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:161]: I meant this bracket is indented one too many.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:417]: This should be `ELTÁVOLÍT` - we use caps for this in all languages as the word is colored red inside `MapSelectionTurnToBan`.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:421]: This should be `VÁLASZT` - we use caps for this in all languages as the word is colored green inside `MapSelectionTurnToPick`.

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:443]: Incorrect indentation.

**Review by nickdnk — CHANGES_REQUESTED** [review]

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [reply translations/hu/get5.phrases.txt]: Another HU translator said we should use `Használd a` instead. I can't make this call, obviously, as I don't speak Hungarian.

**Comment by enerbewow** [comment]
maybe now

**Comment by nickdnk** [comment]
It's better, but you replaced all the spaces with tabs now.

Do search and replace `\t` with `    ` -  that should fix it.

**Comment by enerbewow** [comment]
that `\t `replace to `    ` did not worked for me. so I replaced one by one

**Comment by nickdnk** [comment]
> I go through all the lines again. and fixed every spaces

Much better. But that sounds painful. For future reference, you can just copy a tab symbol (since `\t` didn't work) in your file and search replace it with a space symbol.

I'll wait for review by other HU translators now and when everyone agrees I will merge it.

File: translations/hu/get5.phrases.txt
Comment (Nymu5) [reply translations/hu/get5.phrases.txt]: `Írd be` sounds unsuitable for a command description, but neither would `Használd a` be grammatically correct.
I would suggest using `Használd a {1} parancsot ha nem állsz még készen.`

File: translations/hu/get5.phrases.txt
Comment (Nymu5) [inline translations/hu/get5.phrases.txt:33]: `Használd a {1} parancsot, hogy ha nem állsz még készen.`

File: translations/hu/get5.phrases.txt
Comment (Nymu5) [inline translations/hu/get5.phrases.txt:129]: `Egy admin kezdeményezte ezt az időkérést, ő is fogja tudni folytatni.`

File: translations/hu/get5.phrases.txt
Comment (Nymu5) [inline translations/hu/get5.phrases.txt:221]: `Egy admin szüneteli a mérkőzést.`

File: translations/hu/get5.phrases.txt
Comment (Nymu5) [inline translations/hu/get5.phrases.txt:229]: `Egy admin hajtotta végre a mérkőzés folytatását.`

File: translations/hu/get5.phrases.txt
Comment (Nymu5) [inline translations/hu/get5.phrases.txt:401]: `Használd a {1} <pálya> parancsot, hogy pályát kiválaszd.`

File: translations/hu/get5.phrases.txt
Comment (Nymu5) [inline translations/hu/get5.phrases.txt:405]: Használd a {1} <pálya> parancsot, hogy pályát eltávolítsd.

File: translations/hu/get5.phrases.txt
Comment (Nymu5) [inline translations/hu/get5.phrases.txt:409]: Használd a {1} vagy a {2} parancsot az oldalválasztáshoz.

**Review by Nymu5 — COMMENTED** [review]
I just added some comments on improvement ideas - basically adding `parancsot` for the `Használd a`-term, otherwise just easier language structure in some cases.

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Elfogadva a javaslat. The suggestion is accepted

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Elfogadva a javaslat. The suggestion is accepted

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Elfogadva a javaslat. The suggestion is accepted

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Elfogadva a javaslat. The suggestion is accepted

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Elfogadva a javaslat. The suggestion is accepted

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Elfogadva a javaslat. The suggestion is accepted

File: translations/hu/get5.phrases.txt
Comment (enerbewow) [reply translations/hu/get5.phrases.txt]: Elfogadva a javaslat. The suggestion is accepted

File: translations/hu/get5.phrases.txt
Comment (nickdnk) [inline translations/hu/get5.phrases.txt:425]: Is this right? It's supposed to say "TeamName picked de_dust as map number 2." for instance. It seems this full-stop is in a weird place? @enerbewow @Nymu5  ?

Comment (enerbewow) [reply translations/hu/get5.phrases.txt:425]: No. Sorry I didn't saw it. after `pályát` don't need that dot. 
team_enerbewow most választott pályát 1. pályának - team_enerbewow picked a map for `number of map` map
I actually don't know the text behind {3} but i'll check it.

Comment (enerbewow) [reply translations/hu/get5.phrases.txt:425]: {1} must now {2} a map to play as map {3}.
`team_name` must now `picked` a map to play as map `map_number`

Comment (enerbewow) [reply translations/hu/get5.phrases.txt:425]: so in mine/our translation.
`team_name` most választott egy pályát `map_number` pályának
in raw translate `team_name` now `picked` a map for `map_number` pályának.

**Comment by nickdnk** [comment]
Cool. Looks fine to me now. Can you rebase it on development? You should only have 1 file in your commit. If you are unsure how to do this, just DM me the entire translation file on Discord and I will create a new PR.

**Pull request closed without merging**
